ci/ctest_to_gitlab.sh

```diff
@@ -30,7 +31,7 @@ variables:
   SLURM_EXCLUSIVE: ''
   SLURM_EXACT: ''
   SLURM_CONSTRAINT: $SLURM_CONSTRAINT
-  CRAY_CUDA_MPS: 1
+  CRAY_CUDA_MPS: 0
```

Comment:
Can this be outright removed? Or do you prefer to keep it explicit since it can't be enabled?

Reply:
Want to keep to re-enable on alps.